src/layout/layout.js

```diff
@@ -433,10 +449,8 @@
 
       // Keep an eye on screen size, and add/remove auxiliary class for styling
       // of small screens.
-      this.screenSizeMediaQuery_ = this.matchMedia_(
-          /** @type {string} */ (this.Constant_.MAX_WIDTH));
-      this.screenSizeMediaQuery_.addListener(this.screenSizeHandler_.bind(this));
-      this.screenSizeHandler_();
+
+      screenSizeHandler(this.screenSizeMediaQuery_);
```

Comment:
What are you trying to do here? Doesn't `screenSizeHandler` expect an event? Why are you calling it with a MediaQueryList?

Reply:
My thought was MediaQueryList and MediaQueryListEvent have access to the matches field, so I overloaded the parameter.

Would you prefer we dispatch event instead?

Reply:
I added a better explanation with the JSDoc